e2e/command_test.go

```diff
@@ -467,6 +562,100 @@ func TestE2E_CommandMTLS(t *testing.T) {
 	}
 }
 
+// TestE2E_CommandMTLS_CAPath tests CLI commands using mTLS providing a CAPath rather than a CA cert file. This
+// starts up a local Consul server and runs CTS with TLS in dev mode.
+func TestE2E_CommandMTLS_CAPath(t *testing.T) {
+	t.Parallel()
+
+	srv := newTestConsulServer(t)
+	defer srv.Stop()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "command_mtls_capath")
+
+	tlsc := newMTLSConfig(tempDir)
+	cts := ctsSetupTLS(t, srv, tempDir, dbTask(), tlsc)
+
+	commands := []struct {
+		name           string
+		subcmd         []string
+		outputContains string
+	}{
+		{
+			name:           "task disable",
+			subcmd:         []string{"task", "disable"},
+			outputContains: "disable complete!",
+		},
+		{
+			name:           "task enable",
+			subcmd:         []string{"task", "enable"},
+			outputContains: "Your infrastructure matches the configuration.",
+		},
+	}
+
+	cases := []struct {
+		name         string
+		args         []string
+		envVariables []string
+	}{
+		{
+			name: "ca path flag",
+			args: []string{
+				fmt.Sprintf("-%s=%s", command.FlagHTTPAddr, cts.FullAddress()),
+				fmt.Sprintf("-%s=%s", command.FlagCAPath, tlsc.caPath),
+				fmt.Sprintf("-%s=%s", command.FlagClientCert, defaultCTSClientCert),
+				fmt.Sprintf("-%s=%s", command.FlagClientKey, defaultCTSClientKey),
+				fmt.Sprintf("-%s=%s", command.FlagSSLVerify, "true"),
+				dbTaskName,
+			},
+		},
+		{
+			name: "ca path environment variables",
+			args: []string{
+				dbTaskName,
+			},
+			envVariables: []string{
+				fmt.Sprintf("%s=%s", api.EnvAddress, cts.FullAddress()),
+				fmt.Sprintf("%s=%s", api.EnvTLSCAPath, tlsc.caPath),
+				fmt.Sprintf("%s=%s", api.EnvTLSSSLVerify, "true"),
+				fmt.Sprintf("%s=%s", api.EnvTLSClientCert, defaultCTSClientCert),
```

Comment:
Can we test different client certs on different test cases? It would be good to check that multiple certificates in the CAPath are being used.